src/tcp_connections.rs

```diff
@@ -30,6 +30,9 @@ pub type InTcpStream<T> = Receiver<T>;
 pub type TcpReader<T> = Receiver<T>;
 pub type TcpWriter<T> = OutTcpStream<T>;
 
+/// Magic used to identify a crust packet
+pub const CRUST_PACKET_HEADER : [u8; 4] = [0x63, 0x72, 0x75, 0x31];  // "cru1"
```

Comment:
I think you just need to annotate the initial discovery bootstrap part. I believe the wireshark plugin can tell wireshark that the next step in some sequence is a packet of some protocol on some port, so if it can decode what happens next it simply tells wireshark. Wireshark then should construct a threaded conversation for you.